bindata/network/ovn-kubernetes/managed/alert-rules-control-plane.yaml

```diff
@@ -85,3 +85,13 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: OVNKubernetesOvnKubeMasterLostConnectionToDatabases
+      annotations:
+        summary: ovnkube-master can not connect to the southbound/northbound DB for too long.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/OVNKubernetesOvnKubeMasterLostConnectionToDatabases.md
+        description: |
+          Networking is degraded on nodes when OVNKubeMaster lost connections to SB/NB databases . No networking control plane updates will be applied to the node.
+      expr: |
+        max_over_time(ovnkube_master_libovsdb_monitors[5m]) == 0
```

Comment:
this metric is labelled, so i guess we need to create 2 different alerts for nbdb and sbdb like here https://github.com/openshift/cluster-network-operator/pull/1526/files#diff-3e4b0f06a694a6622c6a7cde162da4c82454271e9c709de8292aa1267dd3f12cR117
And the expected value of this metric is 2, so we probably want to alert when it !=2 or < 2 @dcbw can you help us inderstand what is the best way to create alert based on this monitors metric?

Reply:
does this look any better I am not sure where I can find more on what this metric contains so I can know what to look for

Reply:
also can I use one runbook for both alerts or we need one for each ?

Reply:
runbook for each unfortunately. They have to start with the alert name.